scancodeio/settings/base.py

```diff
@@ -222,6 +222,7 @@
 
 # Celery
 
+CELERY_BROKER_TRANSPORT_OPTIONS = {"socket_timeout": 5, "max_retries": 1}
```

Comment:
Could i make sense to use an environment variable here too?

Reply:
> Is there a way to get a test for this too? or is the setup too complex?

I'm not sure how to make a test case for this. Any ideas?

Reply:
> Could i make sense to use an environment variable here too?

Yes, environment variables can be used to add Broker Configuration.